apps/wallet-dashboard/app/(protected)/vesting/page.tsx

```diff
@@ -142,81 +188,155 @@ function VestingDashboardPage(): JSX.Element {
-                        {vestingSchedule.availableClaiming ? (
-                            <Button onClick={handleCollect}>Collect</Button>
-                        ) : null}
+                <div className="flex flex-col px-lg py-sm">
+                    <div className="flex w-full flex-col items-center justify-center space-y-4 pt-4">
+                        {timelockedStakedObjectsGrouped?.map((timelockedStakedObject) => {
+                            return (
+                                <TimelockedStakedObject
+                                    key={
+                                        timelockedStakedObject.validatorAddress +
+                                        timelockedStakedObject.stakeRequestEpoch +
+                                        timelockedStakedObject.label
+                                    }
+                                    getValidatorByAddress={getValidatorByAddress}
+                                    timelockedStakedObject={timelockedStakedObject}
+                                    handleUnstake={handleUnstake}
+                                />
+                            );
+                        })}
                     </div>
-                )}
-            </div>
+                </div>
+            </Panel>
         </div>
     );
 }
 
+interface TimelockedStakedObjectProps {
+    timelockedStakedObject: TimelockedStakedObjectsGrouped;
+    handleUnstake: (timelockedStakedObject: TimelockedStakedObjectsGrouped) => void;
+    getValidatorByAddress: (validatorAddress: string) => IotaValidatorSummary | undefined;
+}
+function TimelockedStakedObject({
+    getValidatorByAddress,
+    timelockedStakedObject,
+    handleUnstake,
+}: TimelockedStakedObjectProps) {
+    const name =
+        getValidatorByAddress(timelockedStakedObject.validatorAddress)?.name ||
+        timelockedStakedObject.validatorAddress;
+    const sum = timelockedStakedObject.stakes.reduce(
+        (acc, stake) => {
+            const estimatedReward = stake.status === 'Active' ? stake.estimatedReward : 0;
+
+            return {
+                principal: Number(stake.principal) + acc.principal,
+                estimatedReward: Number(estimatedReward) + acc.estimatedReward,
+            };
+        },
+        {
+            principal: 0,
+            estimatedReward: 0,
+        },
+    );
+
+    const [sumPrincipalFormatted, sumPrincipalSymbol] = useFormatCoin(sum.principal, IOTA_TYPE_ARG);
+    const [estimatedRewardFormatted, estimatedRewardSymbol] = useFormatCoin(
+        sum.estimatedReward,
+        IOTA_TYPE_ARG,
+    );
+
+    const supportingText = (() => {
+        if (timelockedStakedObject.stakes.every((s) => s.status === 'Active')) {
+            return {
+                title: 'Estimated Reward',
+                subtitle: `${estimatedRewardFormatted} ${estimatedRewardSymbol}`,
+            };
+        }
+
+        return {
+            title: 'Stake Request Epoch',
```

Comment:
should this be showing up here? in figma, it’s not like that, but maybe I’m missing something? :pray: 
![image](https://github.com/user-attachments/assets/39f04346-66be-41f2-a7a4-5beacc1014f9)

Reply:
yes, if it didnt start earning yet we must display when it will start earning, like we have in the wallet and the wallet dashboard, you can take inspiration from `apps/core/src/components/stake/StakedCard.tsx` https://github.com/iotaledger/iota/blob/46d52c3a5e4ddb57b9779e9e2830fe779c193185/apps/core/src/components/stake/StakedCard.tsx#L72